code/game/objects/items/toys.dm

```diff
@@ -817,6 +817,9 @@
 	item_state = "blahaj"
 	attack_verb = list("gnawed", "gnashed", "chewed")
 
+/obj/item/toy/plushie/shark/explosive
```

Comment:
This makes it so `/obj/random/plushie` can generate these too, since they are not excluded in their spawning list.